server/config/config.go

```diff
@@ -153,13 +153,14 @@ type RemoteExecutionConfig struct {
 }
 
 type ExecutorConfig struct {
-	AppTarget           string `yaml:"app_target" usage:"The GRPC url of a buildbuddy app server."`
-	RootDirectory       string `yaml:"root_directory" usage:"The root directory to use for build files."`
-	LocalCacheDirectory string `yaml:"local_cache_directory" usage:"A local on-disk cache directory."`
-	LocalCacheSizeBytes int64  `yaml:"local_cache_size_bytes" usage:"The maximum size, in bytes, to use for the local on-disk cache"`
-	DockerSocket        string `yaml:"docker_socket" usage:"If set, run execution commands in docker using the provided socket."`
-	DockerNetHost       bool   `yaml:"docker_net_host" usage:"Sets --net=host on the docker command. Intended for local development only."`
-	ContainerdSocket    string `yaml:"containerd_socket" usage:"(UNSTABLE) If set, run execution commands in containerd using the provided socket."`
+	AppTarget             string `yaml:"app_target" usage:"The GRPC url of a buildbuddy app server."`
+	RootDirectory         string `yaml:"root_directory" usage:"The root directory to use for build files."`
+	LocalCacheDirectory   string `yaml:"local_cache_directory" usage:"A local on-disk cache directory."`
+	LocalCacheSizeBytes   int64  `yaml:"local_cache_size_bytes" usage:"The maximum size, in bytes, to use for the local on-disk cache"`
+	DockerSocket          string `yaml:"docker_socket" usage:"If set, run execution commands in docker using the provided socket."`
+	DockerSocketMountPath string `yaml:"docker_socket_mount_path" usage:"If set, mount the configured Docker socket to each action's container at this path, to enable Docker-on-Docker. Intended for backwards compatibility only; use Docker-in-Docker if possible."`
```

Comment:
I'd use the same naming convention that RBE alpha used `docker_sibling_containers`:
https://github.com/bazelbuild/bazel-toolchains/blob/master/rules/exec_properties/exec_properties.bzl#L176

Also it's Docker outside of Docker (DooD)

Also I'd remove the the backwards compatibility part.

Also in order for Docker in Docker (DinD) to be supported, we'd need to support the `--privileged` docker flag - which we don't yet, so I wouldn't put that in the documentation.

Reply:
Done